pool/position.gno

```diff
@@ -11,12 +11,30 @@ import (
 	u256 "gno.land/p/gnoswap/uint256"
 )
 
+var (
+	Q128 = u256.MustFromDecimal(consts.Q128)
+)
+
 // positionGetKey generates a unique key for a position based on the owner's address and the tick range.
 func positionGetKey(
 	owner std.Address,
 	tickLower int32,
 	tickUpper int32,
 ) string {
+	if !owner.IsValid() {
+		panic(addDetailToError(
+			errInvalidAddress,
```

Comment:
we already have `errInvalidInput`.
I think that we use this error code when input parameter check error.

Reply:
Based on the policy for error wording (using detailed error codes), this code does not need to be modified.